pages/web/hosting/configure_ipv6/guide.es-es.md

```diff
@@ -1,74 +1,83 @@
+> Esta traducción ha sido generada de forma automática por nuestro partner SYSTRAN. En algunos casos puede contener términos imprecisos, como en las etiquetas de los botones o los detalles técnicos. En caso de duda, le recomendamos que consulte la versión inglesa o francesa de la guía. Si quiere ayudarnos a mejorar esta traducción, por favor, utilice el botón «Contribuir» de esta página.
 >
 
-**Última actualización: 27/11/2020**
-
-## Introducción
-
-La red de internet funciona desde principios de los años 90 siguiendo la norma IPv4. Esta norma permite proporcionar una dirección IP a cada máquina conectada a la red de Internet: servidores pero también ordenadores, smartphones, tablets y cualquier otro dispositivo conectado a Internet. Esta norma tiene un límite importante: permite identificar algo más de 4.000 millones de dispositivos diferentes. O un dispositivo para dos personas en la Tierra.
+## Objetivo
 
-Pronto se propuso un nuevo protocolo: **IPv6**. Permite identificar más de 340 sextillones de direcciones diferentes. Su despliegue lleva tiempo debido a cambios importantes en toda la red de Internet. 
+La red de internet funciona desde principios de los años 90 siguiendo la norma IPv4. Esta norma permite proporcionar una dirección IP X.X.X (o "X" son números entre 0 y 255) a cada una de las máquinas conectadas a la red de Internet (servidores, ordenadores, smartphones, tabletas, etc.). Sin embargo, esta norma limita el número de dispositivos conectados a Internet a unos 4.000 millones, menos de un dispositivo conectado para dos personas en la Tierra en 2022.
 
-Debido a la escasez de IPv4, cada vez resulta más difícil añadir nuevos recursos a la red de internet. Las conexiones IPv6 solo son útiles si el contenido está también disponible en este protocolo. Así, entre más sitios web tengan IPv6 y más importante será para cada actor de la red migrar a este protocolo.
+Como resultado, se introdujo el protocolo **IPv6** para conectar a la red Internet hasta 340 sextillones de dispositivos. Su implementación toma tiempo porque toda la red de Internet debe integrar esta nueva norma. 
 
-Para más información, consulte el artículo de [Wikipedia](https://es.wikipedia.org/wiki/IPv6){.external} sobre el protocolo IPv6.
+Al estar menos disponibles las direcciones IPv4, es más difícil añadir nuevas máquinas a la red de Internet con la norma IPv4. No obstante, las conexiones con una dirección IPv6 solo son útiles si, por ejemplo, su sitio web también está disponible con el mismo protocolo. De este modo, cuanto más sitios web estén disponibles en IPv6, mayor será el número de actores presentes en la red de internet que transferirán sus dispositivos o máquinas a este nuevo protocolo.
 
-## Objetivo
+Para más información, consulte el artículo de [Wikipedia](https://fr.wikipedia.org/wiki/IPv6){.external} sobre el protocolo IPv6.
```

Comment:
please use the EN wikipedia page for all subs other than FR

Reply:
It placed directly subs links for all subs